docs/devenv_rhel8.md

```diff
@@ -86,6 +87,17 @@ sudo subscription-manager repos --enable rhocp-4.10-for-rhel-8-$(uname -i)-rpms
 sudo dnf install -y cri-o cri-tools
 sudo systemctl enable crio --now
 ```
+
+Download the OpenShift pull secret from the https://console.redhat.com/openshift/downloads#tool-pull-secret page and copy it into the `/etc/crio/openshift-pull-secret` file. 
+
+Run the following commands to configure CRI-O for using the pull secret when fetching container images.
+```bash
+sudo chmod 600 /etc/crio/openshift-pull-secret
+sudo mkdir -p /etc/crio/crio.conf.d/
+sudo cp ~microshift/microshift/packaging/crio.conf.d/microshift.conf /etc/crio/crio.conf.d/
+sudo systemctl restart crio
```

Comment:
After a failed attempt, restarting crio with this command was not enough to reset it for me. I ended up rebooting the VM, but there may be a lighter weight way to force the reset.

Reply:
May be it need `systemctl daemon-reload` and then restart crio service?

Reply:
I added the hack/cleanup.sh call to the instructions, which is cleaning up all the container cache.

For the `systemctl daemon-reload`, I'm not sure it's necessary as we're not changing the configuration file of CRI-O. I guess restarting CRI-O service should cause it rereading its own config, right?